platform/api.progress.nb/src/org/netbeans/api/progress/ProgressHandleFactory.java

```diff
@@ -65,41 +73,62 @@ public static ProgressHandle createHandle(String displayName, Cancellable allowT
     }
 
     /**
-     * Create a progress ui handle for a long lasting task.
+     * Create a progress ui handle for a long lasting task. Since {@code 1.59}, this method was
+     * migrated to {@link ProgressHandle basic Progress API}.
      * @param linkOutput an <code>Action</code> instance that links the running task in the progress bar
      *                   to an output of the task. The action is assumed to open the apropriate component with the task's output.
      * @param displayName to be shown in the progress UI
      * @return an instance of {@link org.netbeans.api.progress.ProgressHandle}, initialized but not started.
+     * @deprecated Please use {@link ProgressHandle#createHandle(java.lang.String, org.openide.util.Cancellable, javax.swing.Action)}
      */
+    @Deprecated
     public static ProgressHandle createHandle(String displayName, Action linkOutput) {
```

Comment:
Please prepare a Jackpot hint that would rewrite to `ProgressHandle.createHandle`.